Source/Cells/BrickCell.swift

```diff
@@ -15,7 +15,7 @@ public protocol AsynchronousResizableCell: class  {
 }
 
 public protocol AsynchronousResizableDelegate: class {
-    func performResize(cell: BrickCell, completion: ((Bool) -> Void)?)
+    func performResize(cell: BrickCell, completion: @escaping (() -> Void))
```

Comment:
Whats the argument for making this not optional anymore

Reply:
I wanted to force people to use the completion block, but it can stay optional.